typescript/infra/src/deploy.ts

```diff
@@ -28,6 +28,11 @@ export async function deployWithArtifacts<T extends HyperlaneFactories>(
     console.error(e);
     contracts = deployer.deployedContracts as any;
   }
-  writeJSON(dir, 'verification.json', deployer.verificationInputs);
+  const existingVerificationInputs = readJSON(dir, 'verification.json');
+  writeJSON(
+    dir,
+    'verification.json',
+    deployer.mergeWithExistingVerificationInputs(existingVerificationInputs),
```

Comment:
What's the advantage of merging instead of re-verifying everything?

Reply:
We don't verify during deployment, and if we didn't merge than we would lose the artifacts